app/views/layouts/application.html.haml

```diff
@@ -20,7 +20,7 @@
           %ul.title-area
             %li.name
               %h1.title
-                %a{href: '/'} <span class="hackedu-logo">hackEDU</span>
+                %a{href: '/'} <span class="hackclub-logo">Hack Club</span>
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace